template/src/utilities/networking.js

```diff
@@ -0,0 +1,80 @@
+import instance from "./axiosInstance";
+
+export function setHeader (property, data) {
+  instance.defaults.headers.common[property] = data;
+};
+
+export function setBaseURL (baseURL) {
+  instance.defaults.baseURL = baseURL;
+};
+
+export function setToken (token, type = null) {
+    switch (type) {
+      case "Paymentez": {
+        return (axios.defaults.headers.common["Auth-Token"] = token);
+      }
+
+      case "Bearer": {
+        return (axios.defaults.headers.common["Authorization"] = `Bearer ${token}`);
+      }
+
+      case "Basic": {
+        const { user, pass } = token;
+        return (axios.defaults.headers.common["Authorization"] = `Basic ${Buffer.from(
+          `${user}:${pass}`
+        ).toString("base64")}`);
+      }
+
+      default: {
+        return (axios.defaults.headers.common["Authorization"] = token);
+      }
+    }
+  };
+
+export function getRequest(api, config) {
+  return instance.get(api, config).then(response => {
```

Comment:
Need to add debug line, e.g __DEV__ && console.log("response>>>>", api, response)